lib/core-common/src/utils/__tests__/normalize-stories.test.ts

```diff
@@ -0,0 +1,75 @@
+import { normalizeStoriesEntry, normalizeStories } from '../normalize-stories';
+
+expect.addSnapshotSerializer({
+  print: (val: any) => JSON.stringify(val, null, 2),
+  test: (val) => typeof val !== 'string',
+});
```

Comment:
TBH I don't really understand why you are using snapshots here, would it not be easier to just use `.toMatchObject()`? I guess it's your call.

Reply:
IMO snapshot tests are easier to maintain